Controllers/AuthenticationController.py

```diff
@@ -24,19 +32,65 @@ def logout():
     AuthenticationService.logout(sessionId)
     return Response("User logged out", status=200)
 
+
 """
 Request params: first name, last name, email (will be used as username), password
 Return: Success or failure codes    
 """
 @auth.route("/signup", methods=["POST"])
 def signup():
-    user = {"firstName": request.form["firstName"], 
-            "lastName": request.form["lastName"], 
+    user = {"firstName": request.form["firstName"],
+            "lastName": request.form["lastName"],
             "email": request.form["email"],
             "password": request.form["password"]
             }
 
     if (not AuthenticationService.signup(user)):
-            return Response(status=400)
-    
-    return Response(status=200) 
\ No newline at end of file
+        return Response(status=400)
+
+    return Response(status=200)
+
+
+@auth.route("/forgot-password", methods=["POST"])
+def forgotPassword():
+    try:
+        userObject = User.objects.get(email=request.form["email"])
+        if userObject == None:
+            return Response("No account with given email found", status=403)
+        else:
+            session = Session(user=userObject)
```

Comment:
Why are we creating a session for the user without authenticating them ?

Reply:
What is the purpose of creating a session here ?

Reply:
I don't think there's connection to verify frontend user session at the moment. So I just checked email. The session is to create a token so that the user may reset their password. I thought about making a different Session class or generating a signed token using a different library but wasn't they are good ideas.